plugin/environment.go

```diff
@@ -473,7 +473,7 @@ func (env *Environment) RunMultiPluginHook(hookRunnerFunc func(hooks Hooks) bool
 }
 
 // performHealthCheck uses the active plugin's supervisor to verify if the plugin has crashed.
```

Comment:
Nit: sense the function name changed
```suggestion
// PerformHealthCheck uses the active plugin's supervisor to verify if the plugin has crashed.
```